extractors/python_functions/keyword_extraction/code_snippet.md

```diff
@@ -1,13 +1,12 @@
 ```python
 import re
 from flashtext import KeywordProcessor
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text"
 YOUR_KEYWORDS: List[str] = ["keyword1", "keyword2", "keyword3"]
 YOUR_LABEL: str = "keyword"
 
-def keyword_extraction(record) -> List[Tuple[str, int, int]]:
+def keyword_extraction(record):
```

Comment:
extraction instead of extractor? is there a pattern we have? if not I would suggest <name_of_module>_extractor / _classifier

generators without _generator since we don't need a visual differentiation in refinery imo since it's an attribute

Reply:
Because all the modules for the endpoint have the suffix "_extraction" I would say we keep and apply the same naming schema to the source code as well!

Reply:
works for me